### ISSUE TITLE
Printer progress object is initialized with incorrect `printTimeOrigin` key instead of `printTimeLeftOrigin`

### ISSUE THREAD

**Issue by ztaylor54** [OctoPrint/OctoPrint#4808]
### The problem

I've come across this while developing a plugin. [According to the data model](https://docs.octoprint.org/en/master/api/datamodel.html#progress-information), the print progress object should contain the key `printTimeLeftOrigin`, but it is instead initialized with the key `printTimeOrigin`:

https://github.com/OctoPrint/OctoPrint/blob/master/src/octoprint/printer/standard.py#L148-L154

And looks like this shortly after OctoPrint starts:

```
{'completion': None, 'filepos': None, 'printTime': None, 'printTimeLeft': None, 'printTimeOrigin': None}
```

After loading a file from the SD card, the [`_updateProgressData()`](https://github.com/OctoPrint/OctoPrint/blob/master/src/octoprint/printer/standard.py#L1089-L1105) function is called, and the incorrect key is overridden with the correct one:

```
{'completion': None, 'filepos': None, 'printTime': None, 'printTimeLeft': None, 'printTimeLeftOrigin': None}
```

This likely doesn't cause any issues for OctoPrint users, but it caused a little headache as a plugin developer when reading the progress object before it had been updated at least once.

### Did the issue persist even in safe mode?

Yes, it did persist

### Version of OctoPrint

main branch

### Operating system running OctoPrint

n/a

### Checklist of files to include below

- [X] Systeminfo Bundle (always include!)
- [ ] Contents of the JavaScript browser console (always include in cases of issues with the user interface)
- [ ] Screenshots and/or videos showing the problem (always include in case of issues with the user interface)
- [ ] GCODE file with which to reproduce (always include in case of issues with GCODE analysis or printing behaviour)

**Comment by ztaylor54** [comment]
Do I need to upload a sysinfo bundle for a code bug? It is irrelevant to this bug report -- you can reproduce it by code inspection.

**Comment by cp2004** [comment]
> Do I need to upload a sysinfo bundle for a code bug? It is irrelevant to this bug report -- you can reproduce it by code inspection.

We can manually approve as well, this does look like a bug of some kind from a brief look at the links you've sent.

**Comment by foosel** [comment]
Fixed by the above commit, will be in 1.10.0
